src/votekit/ballot_graph.py

```diff
@@ -0,0 +1,226 @@
+import matplotlib.pyplot as plt
+import networkx as nx
+from collections import Counter
+from profile import PreferenceProfile
+from distinctipy import get_colors
+from ballot import Ballot
+from typing import Optional, Callable
+from pydantic import BaseModel, validator
+
+
+def build_graph(n):
```

Comment:
worth running `ruff` and `black` on this to bring it in line w/ style guidelines, not going to flag all the issues here but will mention a few worth paying special attention to